### PR TITLE
Add more templates

### PR DESCRIPTION
**Pull request by mclintprojects**
I've added one new email template. I opened this PR so you can look at my commit history and know what's going on with my work. I'll continue adding more templates and once I'm ready I'll ask you to merge it all in.

### CODE REVIEW

**Comment by mclintprojects** [comment]
Fixes issue #31

**Comment by antonreshetov** [comment]
1. Comments do not comply with the [сonvention](https://github.com/antonreshetov/mysigmail/blob/master/.github/COMMIT_CONVENTION.md)
2. You should not use hashtags in commits, as this is a link to issue
3. The templates do not look correct, apparently you have not tested them with different content
4. Template design is questionable

**Comment by mclintprojects** [comment]
@antonreshetov I fixed issue 1 and 2. 
3. Please be more descriptive. What parts of the template do not look correct? I've tested them with all the types of content and the two new ones look and work fine. 
4. The designs were from the link you sent to me and others I found myself. If you have better-looking designs, please share a link. I personally don't think email signatures have to be fancy but that's up to you to decide.

File: src/components/templates/components/Avatar.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
+    <img
+      v-if="image"
```
Comment (antonreshetov) [inline src/components/templates/components/Avatar.vue]: props not exist

Comment (mclintprojects) [reply src/components/templates/components/Avatar.vue]: Thank you. This helped and I've fixed the issue now.

File: src/components/templates/components/Avatar.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
```
Comment (antonreshetov) [inline src/components/templates/components/Avatar.vue]: If you a used wrapper div, you should styled margin it, instead if img. Testing your code please.

Comment (mclintprojects) [reply src/components/templates/components/Avatar.vue]: I see what you mean. Fixed now.

**Comment by mclintprojects** [comment]
Should I add more templates or the two I've added are enough?

**Comment by antonreshetov** [comment]
Template2 not selected wholly via Copy as Select

**Comment by mclintprojects** [comment]
@antonreshetov I didn't know about Copy as Select. I fixed the issue and any new templates I add will not have this problem now that I know of it.

File: src/components/templates/EmailTemplate2.vue [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+                            role="presentation"
+                            style=" margin-top: 5px; margin-left: -2px;"
+                          >
+                            <tbody>
+                              <tr>
+                                <td
+                                  v-for="item in socials.installed"
+                                  :key="item.name"
+                                  align="center"
+                                  style="width: 24px; height: 24px; border-radius: 3px;"
```
Comment (antonreshetov) [inline src/components/templates/EmailTemplate2.vue]: I think that icon should be a 20px like templates 1 & 2

Comment (mclintprojects) [reply src/components/templates/EmailTemplate2.vue]: The design I was looking at had slightly bigger icons that's why I made it 24px. I'll reverse it.

**Comment by mclintprojects** [comment]
I think we can merge these in now?

File: src/components/templates/EmailTemplate2.vue (hunk 3 above)
Comment (antonreshetov) [reply src/components/templates/EmailTemplate2.vue]: Yes, but i think what we need little adapt the design for more consistency with exist template

**Pull request merged**